js_basics/task5-3.js

```diff
@@ -0,0 +1,9 @@
+const length = 40;
+const width = 20;
+const sum = length * width;
```

Comment:
duplicated code, you have the same in previous file

Reply:
Thank you! I made edits for variable name for task5-2.js , changed duplicated code for task5-3.js .